.gitignore

```diff
@@ -62,3 +62,6 @@ typings/
 
 # npm
 package\-lock.json
```

Comment:
I wonder if this pattern works, because I saw another commit that had: `./package-lock.json`, I thought at one point you had to escape the hyphen, but maybe not. (It seems to only be a special character in a "character class." Like [0-9], or [a-z], or whatever.)